plugins/giphy.js

```diff
@@ -13,13 +13,18 @@ module.exports = (function(){
 			}
 
 			if (text.indexOf('giphy:') === 0 && text.length >= 5) {
-				randomGiphy(text.substr(6), function(err, url){
-					if (err){
-						bot.say(to, err);
-					}else{
-						bot.say(to, url);
-					}
-				});
+				var term = text.substr(6);
+				if(term === 'reacharound' || 'reachie' || 'reach-around') {
```

Comment:
That's going to return `true` if term ==='reacharound`, else it'll return `'reachie'` thus the condition will always evaluate to true. I don't think that's what you intended...